docs/source/index.mdx

```diff
@@ -0,0 +1,85 @@
+# Filesystem
+
+🤗 Filesystem is a package that provides a pythonic file interface ([fsspec-compatible](https://filesystem-spec.readthedocs.io/en/latest/)) to the [Hugging Face Hub](https://huggingface.co/). It builds on top of [Hugging Face Hub client library](https://huggingface.co/docs/huggingface_hub/index) to interact with the Hub.
+
+## Installation
+
+```bash
+pip install hffs
+```
+
+## Usage
+
+`HfFileSystem` is the cental class of the library that holds connection information and allows typical filesystem style operations like `cp`, `mv`, `ls`, `du`, `glob`, `get_file`, `put_file` etc.
```

Comment:
Depending on the audience you think will use 🤗 Filesystems, it might help beginners to briefly explain filesystem operations. It may also be nice to add a link where users can see all the supported filesystem operations so they can get a more complete understanding of 🤗 Filesystems capabilities. 

But if this is mostly aimed at more experienced developers or you think it's something users know already (things like `get_file` and `put_file` are mostly new to me), feel free to ignore!

```suggestion
`HfFileSystem` is the library's main class that holds connection information and enables typical filesystem style operations like `cp`, `mv`, `ls`, `du`, `glob`, `get_file`, `put_file` etc.
```